packages/trader/src/Modules/Trading/Containers/strike-param-modal.jsx

```diff
@@ -24,13 +24,13 @@ const StrikeParamModal = ({ is_open, toggleModal, strike, onChange, name, strike
                     is_bubble_hover_enabled
                     message={
                         <Localize
-                            i18n_default_text='<0>{{trade_type}}:</0> You will get a payout if the market is {{payout_status}} this price <0>at the expiry time.</0> Otherwise, your payout will be zero.'
+                            i18n_default_text='If you buy a "<0>{{trade_type}}</0>" option, you receive a payout at expiry if the final price is {{payout_status}} the strike price. Otherwise, your “<0>{{trade_type}}</0>” option will expire worthless.'
```

Comment:
Are we really use the word "worthless"?

Reply:
This is an approved copy by content team and stakeholders, so I guess it is what it is 😂 From clients point of view, it's indeed either you win or you lose everything, so the label does compliment what's happening 🫠